README.md

```diff
@@ -62,6 +62,15 @@ or you can clone the repository locally via
 ```python
 git clone git@github.com:fairlearn/fairlearn.git
 ```
+If you clone from git and wish to use the Fairness dashboard, then you will need to [install Yarn](https://yarnpkg.com/lang/en/docs/install), and then do the following:
+```bash
+cd fairlearn/widget/js
+yarn install
+yarn build:all
+rm -rf dist
```

Comment:
I can write a yarn task to do these removes. These are OS specific, no?

Reply:
I believe that Powershell has the appropriate aliases to make them work like they do in *nix. However, if it could happen automatically in `build:all`, that would be better.

Reply:
As discussed, it's probably safer to just leave these commands there. If the clean ups don't work as-is on another platform, a developer should be able to adapt.

However, per your other suggestion, I have made it plain that the yarn build doesn't have to be done very often